jgrapht-core/src/main/java/org/jgrapht/GraphTests.java

```diff
@@ -599,6 +599,36 @@
         return requireDirectedOrUndirected(graph, GRAPH_MUST_BE_DIRECTED_OR_UNDIRECTED);
     }
 
+    /**
+     * Check if the given graph is triangle free or not
+     * RunTime :- O(|V|*|E|)
```

Comment:
please use math notation, see contributors guidelines in jgrapht wiki

Reply:
@jkinable As you said to start with O(V*E) as the baseline, I used that. There are better algorithms. Should I implement better ones and then send a pull request or accommodate changes you suggested and send a pull request?